### ISSUE TITLE
Change learning rate after loading the pretrained model

### ISSUE THREAD

**Issue by minhson** [tensorpack/tensorpack#1491]
Hello, 
I need to change the learning rate after loading the pretrained model to do finetuning.
Could you help me with this?
Thanks you in advance!

**Comment by ppwwyyxx** [comment]
These are examples: https://github.com/tensorpack/tensorpack/search?q=learning_rate
